test/test_literal/test_duration.py

```diff
@@ -10,37 +10,30 @@
 class TestDuration(unittest.TestCase):
```

Comment:
Not that critical, but you can safely remove this as pytest picks up all classes that start with `Test`
```suggestion
class TestDuration:
```

Reply:
Can of course be done in subsequent PRs, and you may already know, just in case. Think PR is fine as is, already switching to asserts make it much more readable and then makes subsequent PRs smaller and simpler.